lib/client/chart.js

```diff
@@ -178,7 +208,7 @@ function init (client, d3, $) {
   };
 
   // called for initial update and updates for resize
-  chart.update = _.debounce(function debouncedUpdateChart(init) {
+  chart.update = function update(init) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2651207032/issues/source?bid=2452374&fileBranchId=3324571#l211)